gateway/1.4/modules/ROOT/pages/flex-architecture-basic-deployments.adoc

```diff
@@ -0,0 +1,41 @@
+= Basic Deployment Models for Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway supports three basic deployment models, which apply to both Connected Mode and Local Mode:
```

Comment:
Should Connected Mode and Local Mode be lowercase?

Reply:
No, we capitalized these across the docs. Maybe we should change it though.